### PR TITLE
Update mrconfig

### PR DESCRIPTION
**Pull request by deusebio**

### CODE REVIEW

**Review by averma-canonical — APPROVED** [review]
Looks good.

File: myrepos/mrconfig [hunk 1]
```diff
@@ -49,6 +49,12 @@ checkout = git clone 'git@github.com:canonical/kafka-k8s-operator' 'kafka-k8s-op
 [kafka-operator]
 checkout = git clone 'git@github.com:canonical/kafka-operator' 'kafka-operator'
 
+[charmed-kafka-snap]
+checkout = git clone 'git@github.com/canonical/charmed-kafka-snap' 'charmed-kafka-snap'
+
+[charmed-kafka-rock]
```
Comment (welpaolo) [inline myrepos/mrconfig:55]: I see that for the other product we used the suffix `-container`. Shoud we use that ?

**Review by welpaolo — APPROVED** [review]

File: myrepos/mrconfig (hunk 1 above)
Comment (taurus-forever) [reply myrepos/mrconfig:55]: At the moment we are diverged here. Let's commit is AS IS and sync naming separately. Need to ACK with with Mykola.

**Review by taurus-forever — COMMENTED** [review]
Tnx for taking care!
I think you also want to add those new repos into https://github.com/canonical/data-platform/blob/main/.github/repos/settings.yaml

**Comment by deusebio** [comment]
Yes, i'll most definitely do!!!

**Review by taurus-forever — APPROVED** [review]

**Pull request merged**